ridemyway/api/vehicle/models.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.db import models
+
+from ridemyway.api.utilities.base_classes.models import BaseModel
+
+
+class Vehicle(BaseModel):
```

Comment:
I'll also suggest we change this model name to `Car` since `Vehicle` is a very broad name that comprises of all forms of automobiles that can't be used in this app. Good job so far.